A14-channelz.md

```diff
@@ -728,8 +730,27 @@ message ChannelData {
   google.protobuf.Timestamp last_call_started_timestamp = 7;
 }
 
+message ChannelTraceEvent {
+  string description = 1;
+  string error = 2;
+  google.protobuf.Timestamp event_timestamp = 3;
+  ChannelConnectivityState state = 4;
+  // uuid of referenced subchannel.
+  // Optional, only present if this event refers to a child object.
+  string child_uuid = 5; 
```

Comment:
Channels ended up using int64's for the ids.   unless this is the id of the message itself.

Reply:
Done